pkg/skaffold/schema/validation/validation.go

```diff
@@ -56,6 +57,16 @@ func Process(config *latest.SkaffoldConfig) error {
 	return fmt.Errorf(strings.Join(messages, " | "))
 }
 
+// validateTaggingPolicy checks that the tagging policy is valid in combination with other options.
+func validateTaggingPolicy(bc latest.BuildConfig) (errs []error) {
+	if bc.LocalBuild != nil {
+		if bc.LocalBuild.TryImportMissing && bc.TagPolicy.ShaTagger != nil {
+			errs = append(errs, fmt.Errorf("tagging policy 'sha256' can not be used when 'tryImportMissing' is enabled"))
+		}
+	}
+	return
```

Comment:
Nit: this isn't needed

Reply:
This is required because we have a named return value.

Reply:
Oops! 🤦